GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -45,14 +45,28 @@ public void InitRepo()
 
         [TearDown]
         public void TearDownTests()
-        {
-            CmdRunner.DeleteDirectoryWithUnlimitedRetries(this.fastFetchRepoRoot);
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                CmdRunner.DeleteDirectoryWithUnlimitedRetries(this.fastFetchRepoRoot);
+            }
+            else
+            {
+                BashRunner.DeleteDirectoryWithUnlimitedRetries(this.fastFetchRepoRoot);
+            }
         }
 
         [OneTimeTearDown]
         public void DeleteControlRepo()
         {
-            CmdRunner.DeleteDirectoryWithUnlimitedRetries(this.fastFetchControlRoot);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                CmdRunner.DeleteDirectoryWithUnlimitedRetries(this.fastFetchRepoRoot);
+            }
+            else
+            {
+                BashRunner.DeleteDirectoryWithUnlimitedRetries(this.fastFetchRepoRoot);
+            }
```

Comment:
Put this duplicated code in a helper method? Ideally it would be shared with `SharedCacheTests` and `ControlGitRepo` as well (they also have the same code).